ps_linklist.php

```diff
@@ -160,6 +185,27 @@ public function uninstall()
         return $uninstalled && parent::uninstall();
     }
 
+    /**
+     * Migrate data from 1.6 equivalent module (if applicable), then uninstall
+     */
+    private function uninstallPrestaShop16Module()
+    {
+        if (!Module::isInstalled(self::PS_16_EQUIVALENT_MODULE)) {
+            return false;
+        }
+        $oldModule = Module::getInstanceByName(self::PS_16_EQUIVALENT_MODULE);
+        if ($oldModule) {
```

Comment:
What if `$oldModule` is something like `false` or `(int) 0`? We return `true` nonetheless?

Reply:
Yep, we return false only when the module wasn't installed, meaning there is no data to get from it.